products/datacatalog/terraform.yaml

```diff
@@ -155,6 +155,18 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           force_delete: "true"
         oics_vars_overrides:
           force_delete: "true"
+      - !ruby/object:Provider::Terraform::Examples
```

Comment:
Do you think this example is unique enough that it should be in the documentation? If not, I'd recommend doing `skip_docs` for it.

Reply:
Agreed, it is almost identical to one existing case and omitting it